.github/workflows/release-pontos.yml

```diff
@@ -3,24 +3,54 @@ name: Release Python package with pontos
 on:
   pull_request:
     types: [closed]
+  workflow_dispatch:
+    inputs:
+      release-type:
+        type: choice
+        description: What kind of release do you want to do (pontos --release-type argument)?
+        options:
+          - patch
+          - minor
+          - major
+      release-version:
+        type: string
+        description: Set an explicit version, that will overwrite release-type. Fails if version is not compliant.
+
 
 jobs:
   build-and-release:
-    name: Build and release with pontos
-    # If the label 'make release' is set. If PR is closed because of an merge
-    if: contains( github.event.pull_request.labels.*.name, 'make release') && github.event.pull_request.merged == true
-    runs-on: "ubuntu-latest"
+    name: Create a new release
+    # If the event is a workflow_dispatch or on of the labels 'pre release',
+    # 'patch release', 'minor release' or 'major release' is set and PR is
+    # closed because of a merge
+    # NOTE: priority of set labes will be alpha > release-candidate > patch > minor > major,
+    #       so if 'major' and 'patch' labes are set, it will create a patch release.
+    if: |
+      ( github.event_name == 'workflow_dispatch') || (
+        ( contains(github.event.pull_request.labels.*.name, 'alpha release') ||
+          contains(github.event.pull_request.labels.*.name, 'rc release') ||
+          contains(github.event.pull_request.labels.*.name, 'patch release') ||
+          contains(github.event.pull_request.labels.*.name, 'minor release') ||
+          contains(github.event.pull_request.labels.*.name, 'major release')) &&
+          github.event.pull_request.merged == true )
+    runs-on: 'ubuntu-latest'
     steps:
+      - name: Selecting the Release type
+        id: release-type
+        uses: greenbone/actions/release-type@v2
+        with:
+          release-type-input: ${{ inputs.release-type }}
       - name: Release with release action
         uses: greenbone/actions/release@v2
         with:
-          version: "3.10"
-          conventional-commits: true
-          ref: ${{ github.ref_name }}
+          python-version: "3.10"
```

Comment:
Just leave it out. Only set it when a specific Python version is really required here.
```suggestion
```